### DISCUSSION TITLE
Update mathjax default in revealjs

### DISCUSSION THREAD

**Discussion by olivroy** [quarto-dev/quarto-cli#10885] [Feature Requests]
### Description

In RStudio / rmarkdown, the default mathjax was updated to 2.7.9 recently. I thought it may be good to do the same in Quarto. Simply changing this line would probably do the trick?

https://github.com/quarto-dev/quarto-cli/blob/6b363d996e31c65826c2e8186c14a5da6019af96/src/format/reveal/format-reveal.ts#L90

**Comment by cscheid** [comment]
Seems reasonable to me. I created [an issue](https://github.com/quarto-dev/quarto-cli/issues/10887) to track, though @cderv is the expert on our revealjs subsystem and it's his call to make.
